Source/WebCore/platform/graphics/cocoa/GraphicsContextGLCocoa.mm

```diff
@@ -37,12 +37,12 @@
 #import "Logging.h"
 #import "PixelBuffer.h"
 #import "ProcessIdentity.h"
-#import "RuntimeApplicationChecks.h"
 #import <CoreGraphics/CGBitmapContext.h>
 #import <Metal/Metal.h>
 #import <pal/spi/cg/CoreGraphicsSPI.h>
 #import <pal/spi/cocoa/MetalSPI.h>
 #import <wtf/BlockObjCExceptions.h>
+#import <wtf/RuntimeApplicationChecks.h>
```

Comment:
Why not the cocoa one?

Reply:
I could, but the non-Cocoa one is sufficient.